campaigns/mofo-eoy-2016/snippet.html

```diff
@@ -293,7 +293,18 @@
     var snippet = document.getElementById("mofo-eoy-fundraiser");
 
     snippet.addEventListener("show_snippet", function () {
-      setTimeout( function() {
```

Comment:
Dunno, I'm pretty sure I was doing this because older snippets were doing it.

My guess is `show_snippet` was fired too soon, and the elements we wanted to query for were not injected yet, and thus, would fail. Putting them in the `setTimeout` would allow for the next cycle.

It's worth testing.

I would feel a bit better removing it in another patch, just in case it's fixing some fringe bug from before my time?